db/fixtures/chargeback_rates_measures.yml

```diff
@@ -6,8 +6,8 @@
 - :name: Hz Units
   :units_display: ["Hz","KHz","MHz","GHz","THz"]
   :units: ["hertz","kilohertz","megahertz","gigahertz","teraherts"]
-  :step: '1024'
+  :step: '1000'
 - :name: Bytes per Second Units
   :units_display: ["Bps","KBps","MBps","GBps"]
   :units: ["bps","kbps","mbps","gbps"]
-  :step: '1024'
+  :step: '1000'
```

Comment:
@amaurygonzalez  is this `step` somewhere used in calculation of chargeback reports ?